kritis-charts/values.yaml

```diff
@@ -41,7 +41,7 @@ clusterRoleBindingName: kritis-clusterrolebinding
 clusterRoleName: kritis-clusterrole
 cronInterval: 1h
 
-repo: gcr.io/kritis-project/
+repository: gcr.io/kritis-project/
```

Comment:
what is this change for?

Reply:
All of the pod.yamls are configured with [repository](https://github.com/grafeas/kritis/blob/master/kritis-charts/templates/predelete/pod.yaml#L17)

Reply:
yes. @tstromberg found, this piece is broken on master.